.rubocop.yml

```diff
@@ -74,7 +74,5 @@ Style/FormatStringToken:
 Naming/HeredocDelimiterNaming:
   Enabled: false
 
-# Enable check once issue is resolved:
-# https://github.com/bbatsov/rubocop/issues/4940
-Lint/UnneededRequireStatement:
+Style/RedundantReturn:
```

Comment:
Can we also mute cop: `FrozenStringLiteralComment`
On my new setup it complains about it in each and every class/module.
```
test/test_helper.rb:1:1: C: Style/FrozenStringLiteralComment: Missing magic comment # frozen_string_literal: true.
```

Reply:
I have not seen that myself, but updated.